nic_operator_kind/nic_operator_kind_ci_start.sh

```diff
@@ -17,7 +17,7 @@ export KUBECONFIG=${KUBECONFIG:-"/root/.kube/config"}
 
 export KERNEL_VERSION=${KERNEL_VERSION:-$(uname -r)}
 export OS_DISTRO=${OS_DISTRO:-ubuntu}
-export OS_VERSION=${OS_VERSION:-20.04}
+export OS_VERSION=${OS_VERSION:-20.10}
```

Comment:
Please, use this value in common/common_functions.sh too

Reply:
actually this value is different for each CI depending on the environment. So it can not be added as is to the common scripts. 
This value is used to label the node so that the mofed would try to pull the correct value, this started to fail for the kind CI because the function `get_kind_distro_version` is used to determine the mofed container name to upload to kind cluster instead of uploading a static name of the mofed container.

Reply:
It can be changed so that the function `get_kind_distro_version` is used to determine the version. any other suggestion are more than welcome.